src/StatsKit.jl

```diff
@@ -12,6 +12,7 @@ module StatsKit
     @reexport using KernelDensity
     @reexport using Loess
     @reexport using MultivariateStats
+    @reexport using ShiftedArrays
```

Comment:
Shouldn't it be added to REQUIRE / Project.toml as well?

Reply:
Woops, of course. Let's first move to Project.toml: https://github.com/JuliaStats/StatsKit.jl/pull/23

Reply:
Done.